ra/ra.go

```diff
@@ -264,19 +264,45 @@ func validateEmail(ctx context.Context, address string, resolver bdns.DNSClient)
 	return emptyDNSResponseError
 }
 
+// certificateRequestAuthz is a struct for holding information about a valid
+// referenced during a certificateRequestEvent. It holds both the authorization
+// ID and the challenge type that made the authorization valid.
```

Comment:
Maybe let's explain why, of all the fields in the authz, we include the challenge type (e.g. "including the challenge type makes some common analysis easier without looking up the full authz").

Reply:
Good idea. Thank you. Fixed in 3163976